src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import core.basesyntax.domain.Fruit;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverterService {
+    private static final String COMMA_DELIMITER = ",";
+    private static final int COUNT_HEADER_LINES_TO_SKIP = 1;
+    private static final int FRUIT_OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int FRUIT_QUANTITY_INDEX = 2;
+
+    @Override
+    public List<Fruit> convertToFruit(List<String> inputReport) {
```

Comment:
The method implementation should not begin with an empty line. Please remove the redundant empty line at the beginning of the method definition.